### ISSUE TITLE
Update Bio.Application examples in Tutorial to use subprocess directly

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython#3471]
See discussion on #2877, closed via #3344 declaring Bio.Application obsolete, suggesting using subprocess directly.

We need to update the Tutorial to translate the command line wrapper examples to use subprocess directly. Working on Linux/macOS only would be a reasonable first pass but ideally tested as working on Windows too.

The tutorial is written in LaTeX, source here https://github.com/biopython/biopython/tree/master/Doc/Tutorial - on Linux/macOS using the ``make`` command in the ``Doc/`` directory will build this for you.

There is scope for multiple contributors to work on this, most simply by working on different sections of the Tutorial.

**Comment by Caiofcas** [comment]
I was starting to work on removing the `Bio.Application` calls in `chapter_phylo.tex`, but I'm confused if `Bio.Phylo.PAML` was also deprecated, since it seems very different from the other wrappers. In it's wiki page (https://biopython.org/wiki/PAML) and in the Class source code there's also no warning, so I don't know whether it should be removed from the tutorial or not.

It seems that it does provide some value beyond just a CLI wrapper, since it apparently manages the control files as well, and as such might be more troublesome to replace (this is more of a guess as i have not gone deep into what the module does exactly).

TLDR: Should `Bio.Phylo.PAML` also be removed from the tutorial?

**Comment by peterjc** [comment]
The PAML stuff goes beyond just helping to build and run a command line string - it was not intended to be deprecated. So that stays please.
